android/src/main/java/com/stripeterminalreactnative/StripeTerminalReactNativeModule.kt

```diff
@@ -79,10 +78,7 @@ class StripeTerminalReactNativeModule(reactContext: ReactApplicationContext) :
     init {
         context.registerComponentCallbacks(
             object : ComponentCallbacks2 {
-                override fun onTrimMemory(level: Int) {
-                    onTrimMemory(context.applicationContext as Application, level)
-                }
-
+                override fun onTrimMemory(level: Int) {}
                 override fun onLowMemory() {}
                 override fun onConfigurationChanged(p0: Configuration) {}
             })
```

Comment:
nit: you can delete this entire `init { ... }` block now since we're not actually doing anything with these `ComponentCallbacks2`.